### PR TITLE
Add support for include_paths config

### PR DESCRIPTION
**Pull request by jacooobi**
As listed in spec

### CODE REVIEW

**Comment by pbrisbin** [comment]
Can you update the circle.yml to work for forked PRs? See codeclimate-rubocop's as an example https://github.com/codeclimate/codeclimate-rubocop/blob/master/circle.yml

File: Runner.php [hunk 1]
```diff
@@ -20,24 +20,47 @@ public function __construct($config, $server)
...
+        } else {
+            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
+        }
+
+        $this->server->process_work(false);
+    }
+
+    public function queueWithIncludePaths() {
+        foreach ($this->config['include_paths'] as $f) {
+            if ($f !== '.' and $f !== '..') {
```
Comment (pbrisbin) [inline Runner.php:34]: While you're in here -- do you want to fix https://github.com/codeclimate/codeclimate-phpcodesniffer/issues/14 for this engine?

Comment (jacooobi) [inline Runner.php:34]: Sure, no problem

**Comment by ABaldwinHunter** [comment]
@jacobi007 @codeclimate/review 

Anything blocking a merge here?

**Comment by gdiggs** [comment]
LGTM

File: Runner.php [hunk 2]
```diff
@@ -20,24 +20,47 @@ public function __construct($config, $server)
...
+
+                if (is_dir("/code$f")) {
```
Comment (wfleming) [inline Runner.php:36]: I have the same question/concern here as on https://github.com/codeclimate/codeclimate-phpcodesniffer/pull/16/files#diff-65f87d47425b92cdeea437cd228536fdR31

Comment (jacooobi) [inline Runner.php:36]: @wfleming I've tested it and didn't notice any issues

**Comment by mxie** [comment]
Tested this out, and everything looks good.

**Pull request merged**